lib/fathead/flask/output.txt

```diff
@@ -0,0 +1,305 @@
+flask.Blueprint.endpoint	A							http://flask.pocoo.org/docs/0.11/api.html			<section class="prog__container><p>Like Flask.endpoint() but for a blueprint. This does not prefix the endpoint with the blueprint name, this has to be done explicitly by the user of this method. If the endpoint is prefixed with a . it will be registered to the current blueprint, otherwise it’s an application independent endpoint.</p><pre><code>class MyFlask(flask.Flask):\\n  def get_send_file_max_age(self, name):\\n    if name.lower().endswith('.js'):\\n      return 60\\n    return flask.Flask.get_send_file_max_age(self, name)</code></pre></section>	http://flask.pocoo.org/docs/0.11/api.html#flask.Blueprint.endpoint
+flask.Blueprint.errorhandler	A							http://flask.pocoo.org/docs/0.11/api.html			<section class="prog__container><p>Registers an error handler that becomes active for this blueprint only. Please be aware that routing does not happen local to a blueprint so an error handler for 404 usually is not handled by a blueprint unless it is caused inside a view function. Another special case is the 500 internal server error which is always looked up from the application.</p><pre><code>class MyFlask(flask.Flask):\\n  def get_send_file_max_age(self, name):\\n    if name.lower().endswith('.js'):\\n      return 60\\n    return flask.Flask.get_send_file_max_age(self, name)</code></pre></section>	http://flask.pocoo.org/docs/0.11/api.html#flask.Blueprint.errorhandler
+flask.Blueprint.get_send_file_max_age	A							http://flask.pocoo.org/docs/0.11/api.html			<section class="prog__container><p>Provides default cache_timeout for the send_file() functions.</p><pre><code>class MyFlask(flask.Flask):\\n  def get_send_file_max_age(self, name):\\n    if name.lower().endswith('.js'):\\n      return 60\\n    return flask.Flask.get_send_file_max_age(self, name)</code></pre></section>	http://flask.pocoo.org/docs/0.11/api.html#flask.Blueprint.get_send_file_max_age
+flask.Blueprint.has_static_folder	A							http://flask.pocoo.org/docs/0.11/api.html			<section class="prog__container><p>This is True if the package bound object’s container has a folder for static files.</p><pre><code>/myapplication.py\\n/schema.sql\\n/static\\n  /style.css\\n/templates\\n  /layout.html\\n  /index.html</code></pre></section>	http://flask.pocoo.org/docs/0.11/api.html#flask.Blueprint.has_static_folder
+flask.Blueprint.jinja_loader	A							http://flask.pocoo.org/docs/0.11/api.html			<section class="prog__container><p>The Jinja loader for this package bound object.</p><pre><code>/myapplication.py\\n/schema.sql\\n/static\\n  /style.css\\n/templates\\n  /layout.html\\n  /index.html</code></pre></section>	http://flask.pocoo.org/docs/0.11/api.html#flask.Blueprint.jinja_loader
+flask.Blueprint.make_setup_
```

Comment:
Hey, @moollaza , so BS4 produces the unicode string containing '\xbb' i.e.                           >>> Markup("Main &raquo; <em>About</em>").striptags()\nu'Main \xbb About' 
So, when i try to apply the string replace function to escape it , it fails (it fails to find '\xbb'). The only way it works is , if i enclose the above string in triple quotes (then the replace method works), I guess thats a dirty way of doing it ! So please can you suggest some workaround here ? 👍